src/RabbitMQ/AmqpAsyncHandler.php

```diff
@@ -136,7 +136,7 @@ public function daemonize()
             $this->logger->info('Starting AMqpAsyncHandler daemon...');
         }
         
-        while (count($this->channel->callbacks)) {
+        while (true) {
```

Comment:
This is the RabbitMQ recommended way, cause you may want to remove calllbacks.

BTW, it does not really matter for us, so why not !